scripts/perf/test

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+DOWNLOAD_MB=${1:-1000}
+UPLOAD_MB=${2:-0}
+VERSION=${3:-default}
+
+TMP=$(mktemp -d -t s2n-tls-perf-XXXXXXXXXX)
+FILE="${DOWNLOAD_MB}MB-down-${UPLOAD_MB}MB-up-${VERSION}"
+TITLE="${DOWNLOAD_MB}MB Download, ${UPLOAD_MB}MB Upload ${VERSION}"
+DOWNLOAD_B=$(($DOWNLOAD_MB * 1000000))
+UPLOAD_B=$(($UPLOAD_MB * 1000000))
+OUT=target/perf/results
+
+mkdir -p $OUT
+
+perf record \
+  --output $TMP/$FILE.perf \
+  --call-graph dwarf \
+  --event cycles \
+  -- \
+  timeout --signal INT 15 \
+  ./build/bin/s2nd \
```

Comment:
This assumes users have a build directory in their s2n-tls directory, no?

Reply:
correct. it assumes you're following the cmake build instructuions

Reply:
Assuming this will end up in CI, I think we're going to have to make this an argument eventually, but for now, this is probably fine.